Engine/SessionLogs.cs

```diff
@@ -73,6 +74,17 @@ public static void Initialize()
         /// </summary>
         public static void Initialize(string tempLogFileName)
         {
+            // We can't just add this as a parameter with a default value because
+            // it isn't backwards compatible with plugins compiled against older versions.
+            Initialize(tempLogFileName, canDeleteLog);
+        }
+        
+        /// <summary>
+        /// Initializes the logging. 
+        /// </summary>
+        public static void Initialize(string tempLogFileName, bool canDelete)
```

Comment:
Can we add to the xml comment what canDelete does and possibly improve the name maybe 'noWriteLock' or noExclusiveWriteLock?